wagtail/wagtailhalloeditor/editor.py

```diff
@@ -0,0 +1,9 @@
+import json
+
+from wagtail.wagtailcore.fields import BaseRichTextEditor
+
+
+class HalloEditor(BaseRichTextEditor):
```

Comment:
Could we move this to ``wagtailadmin/halloeditor.py`` (and move the static files to somewhere like ``wagtail/wagtailadmin/static_src/wagtailadmin/halloeditor/``)?

Perhaps we should also move the ``insert_editor_js/css`` hooks into methods of the editor class as well to make it easier to kick out hallo js.